extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -557,12 +562,16 @@ export class AmpSlideScroll extends BaseSlides {
       // Set the correct direction.
       direction = direction < 0 ? 1 : -1;
     }
-    this.analyticsEvent_('amp-carousel-change');
+    const vars = {
+      'fromSlide': this.dataSlideIdArr[this.slideIndex_],
+      'toSlide': this.dataSlideIdArr[newSlideIndex],
+    };
+    this.analyticsEvent_('amp-carousel-change', vars);
     // At this point direction can be only +1 or -1.
     if (direction == 1) {
-      this.analyticsEvent_('amp-carousel-next');
+      this.analyticsEvent_('amp-carousel-next', vars);
```

Comment:
For this PR, The idea is to track swipes and changes to carousel. We added specific events called `amp-carousel-next` ,  `amp-carousel-prev` and   `amp-carousel-change`. Next step is to provide the vars associated with these triggers and @camelburrito is adding those vars: `fromSlide` and `toSlide`.